pkgs/development/interpreters/python/build-application.nix

```diff
@@ -0,0 +1,32 @@
+{stdenv}:
+
+{ package
+}:
```

Comment:
This seems like sort of non-standard formatting. Perhaps either `{package}:` like you have above, or 

```
{
  package
}:
```

Not a big deal but this reads kind of weird to me